### PR TITLE
Added my group to the groups.yaml file

### PR DESCRIPTION
**Pull request by joshandetc**
Added my group to the groups.yaml file page

### CODE REVIEW

File: config/groups.yaml [hunk 1]
```diff
@@ -639,3 +639,7 @@
     - "2575138"
     - "2522944"
   id: 190041
+- name: "Group LuNiJo"
+- members:
```
Comment (mbg) [inline config/groups.yaml:643]: This isn't right: each `-` denotes an item in a list, so you have added two items here, but both `name` and `members` are supposed to be part of the same item in the list. 

```suggestion
  members:
```

File: config/groups.yaml [hunk 2]
```diff
@@ -639,3 +639,7 @@
...
+    - "2879998"
+    - "2885512"
```
Comment (mbg) [inline config/groups.yaml:645]: Just to check, are you happy to have only two members in your group? The normal minimum is 3, but I am happy with this if you are.

**Review by mbg — CHANGES_REQUESTED** [review]

**Comment by joshandetc** [comment]
Hi Michael,

Sorry for the late response but are you able to add me to another group? The members in this group are very unresponsive so I'm finding it hard to start the prototype assignment.

Thanks,
Josh

________________________________

**Comment by mbg** [comment]
Superseded by #131

**Pull request closed without merging**